tools/rvn/CommandLineApp.cs

```diff
@@ -364,6 +364,8 @@ private static CommandOption ConfigureServerDirOption(CommandLineApplication cmd
 
         private static void ValidateRavenSystemDir(CommandArgument systemDirArg)
         {
+            var fullPath = systemDirArg.Value.TrimEnd(Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar)
```

Comment:
use DirectoryInfo instead

Reply:
In case of systemDirArg = "C:\Raven_DB\Server\RavenData\System" 
DirectoryInfo returns "C:\Raven_DB\Server\RavenData"
in case of systemDirArg = "C:\Raven_DB\Server\RavenData\System\" 
DirectoryInfo returns "C:\Raven_DB\Server\RavenData\System"

I need System folder in both cases (with/without backslash in the end of systemDirArg).

Reply:
I've meant that after creating a full path you can do:
1. `var targetDirectory = new DirectoryInfo(fullPath)`
2. `if (targetDirectory.Name == "System")` -> you cannot assume that `\` is a valid separator on every platform
3. `if (Path.Combine(targetDirectory.FullName, Voron.Constants.DatabaseFilename))`
4. `if (Directory.Exists(targetDirectory.FullName, "Journals"))`